packages/plugins/cloud/admin/src/utils/api.js

```diff
@@ -0,0 +1,16 @@
+import pluginId from '../pluginId';
+import { request } from '@strapi/helper-plugin';
+
+const verifyIfProjectIsVersionOnGit = async () => {
+  try {
+    const data = await request(
+      `/${pluginId}/verify-project-is-versioned-on-git`,
+      { method: 'GET' }
+    );
+    return data;
+  } catch (error) {
+    return null;
+  }
+};
```

Comment:
I would inline this in the component and it should use the `useFetchClient` hook from `helper-plugin`, not request.